File: In_C_Programming/q13.c
#include <stdio.h>

void get_Merged(int arr[], int arr2[], int temp[], int ln)
{

    for (int i = 0; i < ln; i++)
    {
        temp[i] = arr[i];
    }

    for (int i = 0; i < ln; i++)
    {
        temp[ln + i] = arr2[i];
    }
}

void get_Sorted(int temp[], int ln)
{

    int totalLN = ln + ln;

    for (int i = 0; i < totalLN - 1; i++)
    {
        for (int j = 0; j < totalLN - i - 1; j++)
        {
            if (temp[j] > temp[j + 1])
            {
                temp[j] ^= temp[j + 1];
                temp[j + 1] ^= temp[j];
                temp[j] ^= temp[j + 1];
            }
        }
    }
}

void get_Element(int arr[], int arr2[], int ln)
{

    printf("Enter The Elements of 1st Array: \n");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", &arr[i]);
    }

    printf("Enter The Elements of 2nd Array: \n");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", &arr2[i]);
    }
}

int main()
{

    int size;

    printf("Enter The Aray Length: \n");
    scanf("%d", &size);

    int arr[size];
    int arr2[size];

    int temp[size + size];

    get_Element(arr, arr2, size);

    get_Merged(arr, arr2, temp, size);

    get_Sorted(temp, size);

    printf("The Merged & Sorted Array is: ");

    for (int i = 0; i < size + size; i++)
    {
        printf("%d ", temp[i]);
    }

    printf("\n");

    return 0;
}

File: In_Java_Programming/q13.java
import java.util.Scanner;
import java.util.Vector;

public class q13 {

    public static void get_Sorted(Vector<Integer> vec1, int ln) {

        for (int i = 0; i < ln - 1; i++) {
            for (int j = 0; j < ln - 1 - i; j++) {

                int val1 = j, val2 = j + 1;

                if (val1 > val2) {

                    val1 ^= val2;
                    val2 ^= val1;
                    val1 ^= val2;

                    vec1.set(j, val1);
                    vec1.set(j + 1, val2);
                }
            }
        }
    }

    public static void get_Merged(Vector<Integer> vec1, Vector<Integer> vec2, int ln) {

        for (Integer i : vec2) {
            vec1.addLast(i);
        }
    }

    public static void get_Element(Vector<Integer> vec1, Vector<Integer> vec2, int ln, Scanner sc) {
        System.out.println("Enter The 1st Array Elements: ");

        for (int i = 0; i < ln; i++) {
            int val = sc.nextInt();
            vec1.addLast(val);
        }

        System.out.println("Enter The 2nd Array Elements: ");

        for (int i = 0; i < ln; i++) {
            int val = sc.nextInt();
            vec2.addLast(val);
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter The Array Length: ");
        int ln = sc.nextInt();

        Vector<Integer> vec1 = new Vector<Integer>();
        Vector<Integer> vec2 = new Vector<Integer>();

        get_Element(vec1, vec2, ln, sc);

        get_Merged(vec1, vec2, ln);

        get_Sorted(vec1, ln);

        System.out.println("The Merged and Sorted Array is: ");

        for (Integer i : vec1) {
            System.err.print(i + " ");
        }

        sc.close();
    }
}
